### PR TITLE
Clean the tpm packagegroups for openssl and pkcs#11 and Add Phytec-Esec Startimage

### PR DESCRIPTION
**Pull request by motto-phytec**
1. Remove not necessary tools from the packageroup openssl and pkcs#11
2. Create new tpm provissioning packagegroup with tpm2-tools
3. Create start image for delivery PHYTEC Boards with ESEC IoT Device Manager Platform Support
4. Create rauc Bundle for the start image

### CODE REVIEW

File: recipes-images/packagegroups/packagegroup-provision-tpm.bb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+DESCRIPTION = "Packagegroup for PKCS11 with TPM2."
+LICENSE = "MIT"
+
+inherit packagegroup
+
+SUMMARY_packagegroup-pkcs11-tpm2 = "PKCS11 with TPM 2.0 support"
```
Comment (mschwan-phytec) [inline recipes-images/packagegroups/packagegroup-provision-tpm.bb]: This seems a little weird. Why is the summary of a different packagegroup changed here?

**Review by mschwan-phytec — CHANGES_REQUESTED** [review]

File: recipes-images/packagegroups/packagegroup-provision-tpm.bb (hunk 1 above)
Comment (motto-phytec) [reply recipes-images/packagegroups/packagegroup-provision-tpm.bb]: this was not the intention. is changed

File: recipes-images/packagegroups/packagegroup-provision-tpm.bb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+DESCRIPTION = "Packagegroup for TPM2 provisioning"
+LICENSE = "MIT"
+
+inherit packagegroup
+
+SUMMARY_packagegroup-provision-tpm = "TPM 2.0 provisioning"
```
Comment (mschwan-phytec) [inline recipes-images/packagegroups/packagegroup-provision-tpm.bb]: I think this can also be cleaned up? Just remove the package specific from `SUMMARY`.

**Review by mschwan-phytec — CHANGES_REQUESTED** [review]

**Review by mschwan-phytec — APPROVED** [review]
LGTM

**Pull request merged**